### PR TITLE
Fix issue with Playwright where font wouldn't load or would load incorrectly

### PR DESCRIPTION
**Pull request by EwanFox**
This pr removes the previous font package and adds Noto Sans JP from the Google fonts repository. Tested on a fork and works fine.

### CODE REVIEW

**Comment by djahandarie** [comment]
@EwanFox Awesome, thank you! So excited for the tests to actually be usable now. BTW, could you include this line: https://github.com/microsoft/playwright/issues/18640#issuecomment-1308049819 after the page load but before any screenshots are taken in the JS to make sure that the font is loaded, since it's remote? I just want to make sure we never get any inconsistency

**Comment by EwanFox** [comment]
@djahandarie Done! Added the await before all the screenshotting lines just to be safe.

**Comment by djahandarie** [comment]
@EwanFox Thanks.
* Looks like there's a couple linting errors in CI.
* I took a look at playwright-report generated by the CI, and it seems like "Actual" screenshots are still showing as tofu, so I think the change might not be working entirely correctly? 🤔 Or have I gotten confused about which is "Actual" and which is "Expected"?

**Comment by EwanFox** [comment]
Hey, I will fix the linting errors. I’m pretty sure it still uses the current yml file for workflows until this is merged, but I could be wrong.

**Comment by EwanFox** [comment]
Appears to be a CORS problem, so I'm going to switch it to using a local font.

**Comment by djahandarie** [comment]
@EwanFox Any update on this? Happy to pick it up wherever you left off if you push the latest idea.

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -10,14 +10,17 @@ jobs:
     timeout-minutes: 60
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - name: Remove all fonts
```
Comment (djahandarie) [inline .github/workflows/playwright.yml:13]: I'm surprised latin text renders properly, but I guess it's included in mincho...?

Comment (EwanFox) [reply .github/workflows/playwright.yml:13]: Yes, it is.

File: .github/workflows/playwright.yml [hunk 2]
```diff
@@ -37,7 +40,8 @@ jobs:
       - if: ${{ steps.cache-playwright.outputs.cache-hit != 'true' }}
         name: Install Playwright Browsers
         run: npx playwright install --with-deps chromium
-
+      - name: Check Available Fonts
```
[the remark this answers is not in the document]
Comment (EwanFox) [reply .github/workflows/playwright.yml:43]: Nope, fixing now :)

**Review by djahandarie — APPROVED** [review]

**Pull request merged**